translations.php

```diff
@@ -32,6 +32,7 @@
         - There has been more than one failed login attempt for this account. It is temporarily blocked. <br>
         - Too many failed login attempts from your computer (IP address). This IP address is temporarily blocked. <br> <br>
         To solve the issue, try using different login information, try again later, or <a href=":url">request a new password</a>');
+new TranslatableMarkup("@username doesn't have the required role. You can't add them. Please contact your community manager.");
```

Comment:
There's no need to add translations that are available on the `main` branch to the `translations.php` file. Those translations will be picked up by our translation extraction system automatically.

The `translations.php` file exists for strings that are modified or removed in `main` but are still used in older versions. If we removed them from the codebase entirely, they'd also be removed from our translations system and sites running older versions would not be able to load the translation. By keeping those strings from older versions in `translations.php` we ensure that the translations remain available until all our sites have upgraded past that old version (at which point we can clean up this file).

```suggestion
```

Reply:
Okay, thanks for clarifying, that was very confusing for me. I did it because it was mentioned in the default PR description :)

Reply:
Understandable! We can probably improve the template. If you have ideas on better wording then feel free to open a PR against [the PR template file](https://github.com/goalgorilla/open_social/blob/main/.github/PULL_REQUEST_TEMPLATE.md) :D